assets/js/googlesitekit/modules/create-info-store.js

```diff
@@ -90,10 +90,11 @@ export const createInfoStore = ( slug, {
 				return undefined;
 			}
 
-			const noSetupQueryArgs = ! requiresSetup ? {
-				notification: 'authentication_success',
-				reAuth: undefined,
-			} : {};
+			const noSetupQueryArgs = {};
+			if ( ! requiresSetup && reAuth === true ) {
+				noSetupQueryArgs.notification = 'authentication_success';
+				noSetupQueryArgs.reAuth = undefined;
+			}
```

Comment:
If this fixes something that wasn't working properly, please update `create-info-store.test.js`.

Reply:
Added.